packages/firebase/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@next-auth/firebase-adapter",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "version": "0.0.0",
+  "description": "Firebase adapter for next-auth.",
+  "keywords": [
+    "next-auth",
+    "next.js",
+    "firebase"
+  ],
+  "homepage": "https://next-auth.js.org",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/adapters/issues"
+  },
+  "main": "dist/index.js",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc",
+    "test": "yarn build && jest"
+  },
+  "author": "Ron Houben",
+  "contributors": [
+    "Nico Domino <yo@ndo.dev>"
+  ],
+  "license": "MIT",
+  "peerDependencies": {
+    "next-auth": "^3.1.0"
+  },
```

Comment:
`firebase-admin` should also be under `peerDependencies`

Reply:
`firebase-admin` as a dev and peer dependency then?

Also I've never used it, so feel free if you think you can write some helpful test because I dont think I can :joy: